how-to/finding-reviewers.md

```diff
@@ -1,53 +1,99 @@
+As an editor, you can find reviewers through:
+
+* Suggestions made by the submitting package authors. We suggest only using
+  one of the suggested reviewers if this is the case.
 * Authors of existing [pyOpenSci packages](https://www.pyopensci.org/python-packages/).
-* Other [contributors to pyOpenSci](https://www.pyopensci.org/our-community/).
+* Other [pyOpenSci contributors](https://www.pyopensci.org/our-community/).
 
-When these sources of information are not enough:
+When the above doesn't work and you still need to find a reviewer:
+
+* Check our reviewer sign-up spreadsheet for names of people who have already
+  volunteered to review for pyOpenSci. The link for this document is pinned to
+  the pyOpenSci Slack `#private-editorial-channel`.
 * Ping other editors for ideas.
-* Look for users of the package or the data source/upstream service the package connects to (via their opening issues in the repository, starring it, citing it in papers, talking about it on Twitter).
-* You can also search for authors/maintainers of related packages on [PyPI](https://pypi.org/search/).
-* Post on Twitter and ensure pyOpenSci retweets your post.
+* Post in the pyOpenSci Slack `#software-review` channel.
+* Look for users of the package or the data source/upstream service the
+  package connects to (via their opening issues in the repository, starring
+  it, citing it in papers, talking about it on Twitter).
+* Search for authors/maintainers of related packages on [PyPI](https://pypi.org/search/).
+* Coordinate with our Community Manager to post a call for reviewers on our
+  pyOpenSci social channels.
+
+:::{important}
+
+If you do a reviewer search, please be sure to have new reviewers first sign
+up using our [reviewer signup form](https://docs.google.com/forms/d/e/1FAIpQLSeVf-L_1-jYeO84OvEE8UemEoCmIiD5ddP_aO8S90vb7srADQ/viewform).
+:::
 
 ## Criteria for Choosing Reviewers
 
 Here are criteria to keep in mind when choosing a reviewer. You might need to
-piece this information together by searching `PyPI`, `Conda` / `Conda-forge` and
-the potential reviewer’s GitHub page and general online presence (personal
-website, Twitter).
+piece this information together by searching `PyPI`, `Conda` / `Conda-forge`
+and the potential reviewer’s GitHub page and general online presence (personal
+website, social media profiles).
 
 * Has not reviewed a package for us within the last 6 months.
 * Some package development / contribution experience.
 * Some domain experience in the field of the package or data source.
 * No [conflicts of interest](coi).
 
 (reviewer-diversity)=
-### Reviewer diversity should be prioritized
-Try to balance your sense of the potential reviewer’s experience against the complexity of the package.
+### Reviewer Diversity Should Be Prioritized
+
+Try to balance your sense of the potential reviewer’s experience against the
+complexity of the package.
 
 * **Diversity** - if you have two reviewers, both shouldn’t be cis white males.
```

Comment:
```suggestion
* **Diversity** - if you have two reviewers, only one should be a cis white male.
```

Reply:
bit of a nit, but I'm trying to move my language away from "negatives" and more towards positives. 

related: are we not considering LGBTQIA+ identity as part of diversity?

Reply:
we are definitely not excluding groups in our diversity metrics.